routing/route.hpp

```diff
@@ -72,12 +72,13 @@ class Route
   double GetCurrentDistanceToEndMeters() const;
   double GetMercatorDistanceFromBegin() const;
 
-  void GetCurrentTurn(double & distanceToTurnMeters, turns::TurnItem & turn) const;
+  bool GetCurrentTurn(double & distanceToTurnMeters, turns::TurnItem & turn) const;
   /// @return true if GetNextTurn() returns a valid result in parameters, false otherwise.
-  /// \param distanceToTurnMeters is a distance from current possition to the second turn.
+  /// \param distanceToTurnMeters is a distance from current position to the second turn.
```

Comment:
There are two different functions below: GetNextTurn and GetNextTurns. They have different signature. 
Ok. I'll add a comment for GetNextTurns.